runtime/integration-tests/src/generic/cases/example.rs

```diff
@@ -156,7 +157,21 @@ fn fudge_call_api<T: Runtime + FudgeSupport>() {
 	})
 }
 
+fn pass_time_one_block<T: Runtime>() {
+	let mut env = RuntimeEnv::<T>::from_storage(Default::default());
+
+	let before = env.state(|| pallet_timestamp::Pallet::<T>::get());
+
+	// Not supported in fudge
+	env.pass(Blocks::JumpBySeconds(SECONDS_PER_YEAR));
+
+	let after = env.state(|| pallet_timestamp::Pallet::<T>::get());
+
+	assert_eq!((after - before).into_seconds(), SECONDS_PER_YEAR)
+}
+
```

Comment:
@cdamian I need your fudge wisdom on this.

What I'm doing with the `JumpBySeconds` in the `RuntimeEnv` is jumping to a block in the future where it has passed `SECONDS_PER_YEAR`. This works quite well to "emulate" a state in the future, only creating one block. It's not as accurate as passing all blocks equivalent to a year, but it's much much faster and unlocks these tests for CI. 

Do you think we can modify fudge to add this behavior, jumping to a block in the future without computing the intermediate blocks?

cc @mustermeiszer